tests/ui/async-await/in-trait/warn.stderr

```diff
@@ -0,0 +1,20 @@
+error: use of `async fn` in public traits is discouraged as auto trait bounds cannot be specified
+  --> $DIR/warn.rs:7:5
+   |
+LL |     async fn not_send();
+   |     ^^^^^
+   |
+   = note: you can suppress this lint if you plan to use the trait locally, for concrete types, or do not care about auto traits like `Send` on the `Future`
+note: the lint level is defined here
+  --> $DIR/warn.rs:4:9
+   |
+LL | #![deny(async_fn_in_trait)]
+   |         ^^^^^^^^^^^^^^^^^
+help: you can alternatively desugar to a normal `fn` that returns `impl Future` and add any desired bounds such as `Send`
+   |
+LL -     async fn not_send();
+LL +     fn not_send() -> impl std::future::Future<Output = ()> + Send;
+   |
```

Comment:
I think this should probably go first, above the note about suppressing.

Reply:
This is not possible. By construction, we print all suggestions after notes:

https://github.com/rust-lang/rust/blob/2e5a9dd6c9eaa42f0684b4b760bd68fc27cbe51b/compiler/rustc_errors/src/emitter.rs#L2113-L2161

Reply:
+1.

Reply:
It's not the first time I've been annoyed by that, but oh well.